File: mvc-server/src/main/java/com/yosep/virtual_thread_vs_webflux_bench_mark_main/db/repository/TestDataRepository.java
package com.yosep.virtual_thread_vs_webflux_bench_mark_main.db.repository;

import com.yosep.virtual_thread_vs_webflux_bench_mark_main.db.entity.TestData;
import org.springframework.data.jpa.repository.JpaRepository;

public interface TestDataRepository extends JpaRepository<TestData, Long> {
}

File: webflux-server/src/main/kotlin/com/yosep/virtual_thread_vs_webflux_bench_mark_main/db/repository/TestDataRepository.kt
package com.yosep.virtual_thread_vs_webflux_bench_mark_main.db.repository

import com.yosep.virtual_thread_vs_webflux_bench_mark_main.db.entity.TestData
import org.springframework.data.repository.reactive.ReactiveCrudRepository

interface TestDataRepository : ReactiveCrudRepository<TestData, Long>
